app/bundles/DashboardBundle/Config/config.php

```diff
@@ -77,6 +78,14 @@
                     'session',
                 ],
             ],
+            'mautic.dashboard.widget_detail.event.factory' => [
+                'class'     => \Mautic\DashboardBundle\Widget\WidgetDetailEventFactory::class,
+                'arguments' => [
+                    'translator',
+                    'mautic.cache.provider',
+                    'mautic.helper.user',
```

Comment:
Here could be on third place 'mautic.security' service
I tried push it but do not have permission

Reply:
@kuzmany I don't have time to fix it now. But I agree with you on that.